websites/B/BanG Dream! Wikia/dist/metadata.json

```
@@ -5,10 +5,11 @@
   },
   "service": "BanG Dream! Wikia",
   "description": {
-    "en": "Presence for the English Wikia of the BanG Dream! series"
+    "en": "Presence for the English Wikia of the BanG Dream! series",
+    "ga_IE": "Láithreacht don Wikia Béarla an aisling Bang!sraith"
   },
```

Shouldn't there be a space after the !
```suggestion
    "ga_IE": "Láithreacht don Wikia Béarla an aisling Bang! sraith"
  },
```